### ISSUE TITLE
[Bug]: Ring notification on dual doorbell is not processed

### ISSUE THREAD

**Issue by thieren** [bropat/eufy-security-client#223]
### Client version

2.2.0-rc.2

### Node version

all

### Operating System type

Other

### Operating system version

all

### Describe the bug

As described [here](https://github.com/homebridge-eufy-security/plugin/issues/167) the ring notification is not processed (and therefore the appropriate event is not emitted) by the library for certain doorbells.

As can be seen in the logs the client receives various push messages in sequence: motion, person detected and ring. But the `push_count` is increased by each notification, so the ring notification hast a value of 3:

```
2022-09-22 04:33:28.802  DEBUG [EufySecurity.onPushMessage] Received push message 
{
  name: 'Front Doorbell Camera',
  event_time: 1663821202778,
  type: 93,
  station_sn: 'T8203P11xxxxx',
  device_sn: 'T8203P11xxxxx',
  title: 'Video Doorbell Dual (Wired)',
  content: 'Someone has press doorbell',
  push_time: 1663821208682,
  person_name: '',
  channel: 0,
  cipher: 62,
  event_session: '20220921_233322',
  event_type: 3103,
  file_path: '/media/mmcblk0p1/Camera00//userdata/Camera00/20220921233322.dat.dat',
  pic_url: ***
  push_count: 3,
  notification_style: 3
}
```

Since the notification will only be processed if this parameter is `1` or `undefinded` the client will not process this notification.

https://github.com/bropat/eufy-security-client/blob/c9f8f2c4f37d8fcb3b3ca40aaa6c0c7fe4750dc3/src/http/device.ts#L1639

I'd assume the `push_count` check is to avoid multiple notifications of the same kind? Is there an example of when this check is useful - or phrased the other way around: why was this check introduced?

### To reproduce

2. Click on '...'

### Screenshots & Logfiles

[eufy-log.log](https://github.com/bropat/eufy-security-client/files/9625179/eufy-log.log)

**Comment by bropat** [comment]
@thieren 

> I'd assume the `push_count` check is to avoid multiple notifications of the same kind?

Correct.

> Is there an example of when this check is useful - or phrased the other way around: why was this check introduced?

Depending on the notification type setting (`MOST_EFFICIENT = 1, INCLUDE_THUMBNAIL = 2, FULL_EFFECT = 3`), multiple notifications arrive for the same event. Specifically, `INCLUDE_THUMBNAIL` first sends a notification of the event and then a second notification sends the image. This is why this check was introduced.
What about the Dual Doorbell when the notification type is set to `INCLUDE_THUMBNAIL`?

Translated with www.DeepL.com/Translator (free version)

**Comment by bropat** [comment]
Fixed in version `2.2.0`.
